### PR TITLE
Use existing secret for OAUTH2_PROXY_CLIENT_ID, OAUTH2_PROXY_CLIENT_SECRET and OAUTH2_PROXY_COOKIE_SECRET variables

### PR DESCRIPTION
**Pull request by lewandowskim1988**
### Description of the change

This change introduce possibility to use existing secret where you can put `OAUTH2_PROXY_CLIENT_ID`, `OAUTH2_PROXY_CLIENT_SECRET` and `OAUTH2_PROXY_COOKIE_SECRET` variables.

### Benefits

`OAUTH2_PROXY_CLIENT_SECRET` value is something that should not be push to git repo as plaintext so it is better to use some existing secret where this value is provided.
You can also use some secret operator like `external-secrets` to fetch those values from some secure secret store.

### Possible drawbacks

N/A

### Applicable issues

N/A

- fixes #

### Additional information

I tested this change on my local k8s cluster without any issues.

### CODE REVIEW

**Review by absoludity — APPROVED** [review]
Thanks @lewandowskim1988 , yes, great option to refer to the required auth proxy secret instead of being forced to specify the secrets in the values. Excellent.

**Comment by absoludity** [comment]
Also, note that if you propose this change directly to the upstream Bitnami Kubeapps chart, at https://github.com/bitnami/charts/ , it may be available much sooner (as a new chart version is released once a PR is approved, as opposed to in our repo, where we have the development version of the chart, which will wait until we release a new app version and sync the upstream chart with our dev changes as part of that).

**Comment by lewandowskim1988** [comment]
I am closing this pull request since I moved this change to https://github.com/bitnami/charts/.
[PR](https://github.com/bitnami/charts/pull/14072) link.

**Pull request closed without merging**
